src/core/endpoints/objects/channel/get_all.ts

```diff
@@ -90,9 +90,10 @@ export class GetAllChannelsMetadataRequest<
 
   protected get queryParameters(): Query {
     const { include, page, filter, sort, limit } = this.parameters;
-    const sorting = Object.entries(sort ?? {}).map(([option, order]) =>
-      order !== null ? `${option}:${order}` : option,
-    );
+    let sorting: string | string[] = '';
```

Comment:
👍🏻 
I think we may need to update js sdk doc [here](https://www.pubnub.com/docs/sdks/javascript/api-reference/objects#get-metadata-for-all-channels).

I see that [openapi](https://github.com/pubnub/openapi/blob/master/Specifications/Objects/objects.yaml#L699) says possible value is string.